akka-docs/src/main/paradox/persistence-query.md

```diff
@@ -27,6 +27,10 @@ side of an application, however it can help to migrate data from the write side
 simple scenarios Persistence Query may be powerful enough to fulfill the query needs of your app, however we highly
 recommend (in the spirit of CQRS) of splitting up the write/read sides into separate datastores as the need arises.
 
+
+The [CQRS with Akka 2.6 video](https://akka.io/blog/news/2020/02/05/akka-cqrs-video) is a good starting point for
+learning how to use `eventsByTag` to implement CQRS with Akka.
```

Comment:
Maybe a link back to the event sourcing video?

Reply:
added https://github.com/akka/akka/pull/28575/commits/e4c840cce205c52838cf7e1a49e7cb612ef77ca9